relay-server/src/services/store.rs

```diff
@@ -507,6 +507,7 @@ impl StoreService {
             BucketViewValue::Gauge(g) => MetricValue::Gauge(g),
         };
 
+        // TODO: propagate the `received_at` field upstream.
```

Comment:
In a follow up PR we want to propagate to kafka the message with the received at field.

Reply:
Let's put the gh issue here